msal/application.py

```diff
@@ -21,13 +21,14 @@
 import msal.telemetry
 from .region import _detect_region
 from .throttled_http_client import ThrottledHttpClient
+from .cloudshell import _is_running_in_cloud_shell
 
 
 # The __init__.py will import this. Not the other way around.
-__version__ = "1.17.0"  # When releasing, also check and bump our dependencies's versions if needed
+__version__ = "1.18.0b1"  # When releasing, also check and bump our dependencies's versions if needed
```

Comment:
@jiasli , usually this is the only thing that I need to change for a new release.

So, this time we will go with "1.180**b1**", and it should just work?

Reply:
This, it will!

Reply:
Oh, maybe changelog/history notes?

Reply:
No, we don't have that. What we did is to copy the each release PR's description as release notes. And then all release notes become the history. https://github.com/AzureAD/microsoft-authentication-library-for-python/releases